tests/cli.rs

```diff
@@ -83,6 +83,12 @@ fn get_base_command() -> Command {
     }
     cmd.env("HOME", "");
     cmd.env("NETRC", "");
+    cmd.env(
+        "XH_CONFIG_DIR",
+        dirs::config_dir()
```

Comment:
Would just blanking it like the others work? Or is looking at `~/.config/xh-test` useful for something?

Reply:
Ah, I see what you mean. Let me check if that works...